kubejs/server_scripts/tfg/recipes.nuclear.js

```diff
@@ -47,13 +47,20 @@ function registerTFGNuclearRecipes(event) {
 
 	//#region Mars Water
 
-	event.recipes.gtceu.electrolyzer('mars_water')
-		.inputFluids(Fluid.of('tfg:semiheavy_ammoniacal_water', 1000))
+	event.recipes.gtceu.electrolyzer('mars_heavy_water')
```

Comment:
I wonder if this should be a centrifuge recipe instead, since it's similar to how you get salt out of sea water -- I'd expect an electrolyzer recipe to return the hydrogen, oxygen, deuterium etc instead